packages/fether-electron/package.json

```diff
@@ -25,6 +25,7 @@
   ],
   "homepage": "https://github.com/paritytech/fether",
   "parity": {
+    "version": ">=2.4.1",
```

Comment:
I think this is too lenient: if in 1y, someone uses parity-ethereum 3.x.x with this version of fether, it'll probably not work.

My initial idea was to put an exact version here. But then for people running nodes separately in a terminal, it'll be a headache.

We could probably do a compromise like `~2.4.1`? I'm not sure. Anyways, we can still discuss this after the PR gets merged.

Reply:
I agree. Setting it to `^2.4.1` for now but open to discussing / modifying after this PR.